mlxtend/evaluate/__init__.py

```diff
@@ -40,4 +41,5 @@
            "RandomHoldoutSplit", "PredefinedHoldoutSplit",
            "ftest", "combined_ftest_5x2cv",
            "proportion_difference", "bias_variance_decomp",
-           "accuracy_score", "create_counterfactual"]
+           "accuracy_score", "create_counterfactual",
+           "time_series"]
```

Comment:
I think `"time_series"` should be `"GroupTimeSeriesSplit"` here

Reply:
Yes I was wrong, corrected and commited changes.